README.rst

```diff
@@ -340,206 +342,202 @@ For example:
 
 .. code-block:: lua
 
-    print("Hi, I am {{ .Name }} application")
+    print("Hi, I am the {{ .Name }} application")
     print("I also have a stateboard named {{ .StateboardName }}")
 
-.. _cartridge-cli-building-an-application:
+.. _cartridge-cli-building-the-application:
 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Building an application
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Building the application
+~~~~~~~~~~~~~~~~~~~~~~~~
 
-To build your application locally (for local testing), say this in any directory:
+To build your application locally (for local testing), run this in any directory:
 
 .. code-block:: bash
 
     cartridge build [PATH] [flags]
 
 The following options (``[flags]``) are supported:
 
-* ``--spec`` is the path to rockspec to use for the current build.
-  *Note*, that rockspec file name should be in lowercase.
+* ``--spec`` is the path to the ``.rockspec`` to use for the current build.
+  *Note* that the ``.rockspec`` file name should be in lowercase.
 
-.. // Please, update the doc in cli/commands on updating this section
+.. // Please update the doc in cli/commands on updating this section
 
-This command requires one argument — the path to your application directory
-(i.e. to the build source). The default path is ``.`` (the current directory).
+The command requires one argument---the path to your application directory
+(that is, to the build source).
+The default path is ``.`` (the current directory).
 
-This command runs:
+``cartridge build`` is executed in two steps:
 
-1. ``cartridge.pre-build`` if the
-   `pre-build file <Special files_>`_ exists.
-   This builds the application in the ``[PATH]`` directory.
-2. ``tarantoolctl rocks make`` if the
-   `rockspec file <Special files_>`_ exists.
-   This installs all Lua rocks to the ``[PATH]`` directory.
+1. If there is a  :ref:`pre-build file <cartridge-cli-special-files>`, ``cartridge.pre-build`` builds the application in the ``[PATH]`` directory.
```

Comment:
Can we name it somehow different? Say, `DESTDIR`? I read it several times until got that it is the `PATH` from the command declaration above, not how it is usually used (the environment variable with executables paths).

Reply:
'cartridge.pre-build build the application' — it looks as a mistake.

Reply:
So, we call the prebuild script and then build the app in the DESTDIR. Correct? If so, please, change the description. Now I cannot undesrtand it.